frontend/src/router.jsx

```diff
@@ -1,27 +1,70 @@
 import React from "react";
 import HomePage from "./containers/HomePage";
 import RestautantListPage from "./containers/RestaurantListPage";
-import MyRestaurantsPage from './containers/MyRestaurantsPage';
+import MyRestaurantsPage from "./containers/MyRestaurantsPage";
 import RestaurantsMapPage from "./containers/RestaurantsMapPage";
 import RestaurantPage from "./containers/RastaurantPage";
 import { Route, Switch } from "react-router-dom";
 import LogInPage from "./containers/LogInPage";
 import SignUpPage from "./containers/SignUpPage";
+import Profile from "./containers/Profile";
 import UserInfoPage from "./containers/UserInfo";
 
+const routes = [
+  {
+    path: "/",
+    component: HomePage
+  },
+  {
+    path: "/restaurants",
+    component: RestautantListPage
+  },
+  {
+    path: "/restaurants/:id",
+    component: RestaurantPage
+  },
+  {
+    path: "/profile/my_restaurant/:id",
+    component: RestaurantPage
+  },
+  {
+    path: "/restaurants-map",
+    component: RestaurantsMapPage
+  },
+  {
+    path: "/log-in",
+    component: LogInPage
+  },
+  {
+    path: "/sign-up",
+    component: SignUpPage
+  },
+  {
+    path: "/profile",
+    component: Profile
+  },
+  {
+    path: "/profile/persona_info",
+    component: UserInfoPage
+  },
+  {
+    path: "/profile/current_orders",
+    component: Profile
+  },
+  {
+    path: "/profile/my_restaurants",
+    component: MyRestaurantsPage
+  }
+];
+
 class Router extends React.Component {
   render() {
     return (
       <React.Fragment>
         <Switch>
-          <Route exact path="/" component={HomePage} />
-          <Route path="/restaurants" exact component={RestautantListPage} />
-          <Route path="/restaurants-map" exact component={RestaurantsMapPage} />
-          <Route path="/restaurants/:id" exact component={RestaurantPage} />
-          <Route path="/log-in" component={LogInPage} />
-          <Route path="/sign-up" component={SignUpPage} />
-          <Route path="/profile" exact component={UserInfoPage} />
-          <Route path="/profile/my-restaurants" exact component={MyRestaurantsPage} />
+          {routes.map(({ path, component }) => (
```

Comment:
If not, then I prefer to read the code in the old format

Reply:
DRY principle, more clean component for me, and better readability in my opinion. 
Yes, it possible to make exact prop optional.
However, in case that you prefer previous variant, i will change it

Reply:
The code is written correctly. This question like "What is better adidas or nike?" The old format had 8 lines of code. 
In the new format you create a new array, use map in Switch and we get 48 lines of code. But if we be able to get this array from another resource(from api or constants folder  ) then i agree with you.